diff --git a/CMakeLists.txt b/CMakeLists.txt
index d6f9e68..8717d07 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,10 +87,10 @@ endif()
 if(MSVC)
   # Microsoft Visual Studio Compiler
   if(NOT DEFINED ENV{CMAKE_BUILD_TYPE})
-    # Possible values are: Debug, Release, RelWithDebInfo or MinSizeRel    
+    # Possible values are: Debug, Release, RelWithDebInfo or MinSizeRel
     set(CMAKE_BUILD_TYPE RelWithDebInfo)
     message(STATUS "No Configuration Type specified, use default: ${CMAKE_BUILD_TYPE}")
-  else()    
+  else()
     message(STATUS "Configuration Type was specified: ${CMAKE_BUILD_TYPE}")
   endif()
 endif()
